common/sgx/rand.S

```diff
@@ -3,15 +3,38 @@
 
 .text
 .globl  oe_rdrand
+.type oe_rdrand, @function
+.globl  oe_rdseed
+.type oe_rdseed, @function
 
 oe_rdrand:
+.cfi_startproc
```

Comment:
Since the function does not use the stack, there is no need for rbp, rsp manipulation as well as the leave instruction.

Reply:
Removed from both versions.